### DISCUSSION TITLE
Shedule cron at a specific hour ?

### DISCUSSION THREAD

**Discussion by rodinux** [prodrigestivill/docker-postgres-backup-local#134] [Q&A]
Hello, thanks for this code. I would like to know if it is possible to run the SHEDULE at a specific hour ? I can't found crontab command when connected in the container... 
Can we use something like `SCHEDULE=@Scheduled(cron = "0 0 12 * * ?")` ?

**Comment by rodinux** [comment]
or can it be a full crontab ?
 `SCHEDULE= "0 0 12 * * *"`

**Comment by rodinux** [comment]
> Hello, thanks for this code. I would like to know if it is possible to run the SHEDULE at a specific hour ? I can't found crontab command when connected in the container... Can we use something like `SCHEDULE=@Scheduled(cron = "0 0 12 * * ?")` ?

this don't work...

**Reply by rodinux** [reply]
go-cron use also seconds we need have something like `* * * * * *`  and we fund with quotes it does not match...

**Reply by prodrigestivill** [reply]
go-cron used by this project is configured so that the seconds are optional https://github.com/prodrigestivill/go-cron/commit/0395b7f260996f27bd493d35898a5647cd98ed28

**Reply by rodinux** [reply]
Ok, so If I try like you suggest `SCHEDULE="5 * * * *"` to verify if it will works also

**Reply by rodinux** [reply]
I think the quotes do a mismatch

**Reply by rodinux** [reply]
I do another try beacause I was wrong with this  `SCHEDULE="*/5  * * * *"`

**Reply by rodinux** [reply]
I confirm with quotes it does not work, now I try `SCHEDULE=*/5  * * * *` to see what happens

**Reply by rodinux** [reply]
It works, conclusion it works without quotes !

**Comment by prodrigestivill** [comment]
You should use cron format  as states the README file.

To test your variable you can use [crontab.guru](https://crontab.guru/#0_12_*_*_*) .

```
SCHEDULE="0 12 * * *"
```

**Reply by rodinux** [reply]
go-cron use also seconds we need have something like `* * * * * *` and we fund with quotes it does not match... it works with `SHEDULE=0 0 12 * * *` for example

**Reply by rodinux** [reply]
I think the quotes do a mismatch

**Reply by prodrigestivill** [reply]
quotes or not depends on the place you are setting it up, lookup to the documentation of the platform you are using (docker-compose, helm, etc...)

**Comment by rodinux** [comment]
I think I found a solution `- SCHEDULE=0 0 19 * * *` for example, we have do a test with every 3 minutes with quotes it does not work  but like this it did the staff  `- SCHEDULE=0 */3 * * * *`
